event-log/src/main/scala/io/renku/eventlog/subscriptions/newEvent/ProjectPrioritisation.scala

```diff
@@ -26,10 +26,10 @@ import ch.datascience.tinytypes.{BigDecimalTinyType, TinyTypeFactory}
 import eu.timepit.refined.api.Refined
 import eu.timepit.refined.numeric.NonNegative
 import io.renku.eventlog.EventDate
-import io.renku.eventlog.subscriptions.ProjectPrioritisation.{Priority, ProjectIdAndPath, ProjectInfo}
+import io.renku.eventlog.subscriptions.newEvent.ProjectPrioritisation.{Priority, ProjectIdAndPath, ProjectInfo}
 
-private class ProjectPrioritisation {
-  import Priority._
+private[subscriptions] class ProjectPrioritisation {
```

Comment:
Why do we need to make it private for the `subscription`?

Reply:
Yes good catch it shouldn't be used outside of the newandrecoverable package